CMakeLists.txt

```diff
@@ -17,23 +17,20 @@ ELSE()
 ENDIF()
 
 # The source of the library we want to build
-add_library(${PROJECT_NAME} src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
+add_library(${PROJECT_NAME} SHARED src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
```

Comment:
This makes the library always build as a shared library (.so in linux, .dylib in macos)